modules/user-access/pages/access-control-model.adoc

```diff
@@ -145,4 +149,110 @@ By default, both ACL entries (`READ` and `EXECUTE`) for a query are _unspecified
 ** Even if a user has the `READ_QUERY` permission on a graph, they cannot read a query if the `READ` ACL entry for the query is specified and the user is not on the list.
 
 .How permission is evaluated for a query operation
-image::acl-query-flow.svg[]
\ No newline at end of file
+image::acl-query-flow.svg[]
+
+=== Example 1: `READ` Privilege
+
+In the following example, `user1` protects their query `q1` from being seen by anyone, including superusers.
```

Comment:
"from being seen by anyone, including superusers."
This is a new behavior: before, superusers could do anything.  It seems we should put this in a note when talking about the superuser role.